### DISCUSSION TITLE
我这里提示要更新 node  我怎么在 actions 里更新node?

### DISCUSSION THREAD

**Discussion by iibrat** [tw93/Pake#658] [Q&A] [answered]
我这里提示要更新 node  我怎么在 actions 里更新node?
[windows-latest](https://github.com/iibrat/Pake/actions/runs/7765179592/job/21179522489)
Node.js 16 actions are deprecated. Please update the following actions to use Node.js 20: actions/checkout@v3, actions/setup-node@v3, actions/cache/restore@v3, actions/upload-artifact@v3, actions/cache/save@v3. For more information see: https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/.

**Comment by tw93** [comment, marked as answer]
这个可以不管，其实是actions最近的一个更新，我最近更新一下.github里面的版本就好了，这个不影响使用，只不过强迫症有些受不了（我也是强迫症
